src/Spain/Mews.Fiscalizations.Spain.Tests/Basics.cs

```diff
@@ -10,25 +10,43 @@ namespace Mews.Fiscalizations.Spain.Tests.IssuedInvoices;
 
 public class Basics
 {
-    public static readonly X509Certificate2 Certificate = new X509Certificate2(
+    private static readonly X509Certificate2 Certificate = new(
         rawData: Convert.FromBase64String(System.Environment.GetEnvironmentVariable("spanish_certificate_data") ?? "INSERT_CERTIFICATE_DATA"),
         password: System.Environment.GetEnvironmentVariable("spanish_certificate_password") ?? "INSERT_CERTIFICATE_PASSWORD",
         keyStorageFlags: X509KeyStorageFlags.DefaultKeySet
     );
-    public static readonly Client Client = new Client(Certificate, Environment.Test, httpTimeout: TimeSpan.FromSeconds(30));
 
-    public static readonly LocalCounterParty Issuer = LocalCounterParty.Create(
+    private static readonly Client Client = new(Certificate, Environment.Test, httpTimeout: TimeSpan.FromSeconds(30));
+
+    private static readonly LocalCounterParty Issuer = LocalCounterParty.Create(
         name: Name.CreateUnsafe("Issuing company"),
         nifVat: System.Environment.GetEnvironmentVariable("spanish_issuer_tax_number") ?? "INSERT_ISSUER_TAX_NUMBER"
     ).Success.Get();
 
-    public static readonly LocalCounterParty ReceivingCompany = LocalCounterParty.Create(
+    private static readonly LocalCounterParty ReceivingCompany = LocalCounterParty.Create(
         name: Name.CreateUnsafe("Receiving company"),
         nifVat: System.Environment.GetEnvironmentVariable("spanish_receiver_tax_number") ?? "INSERT_RECEIVER_TAX_NUMBER"
     ).Success.Get();
 
+    private static readonly TaxExemptItem[] UntaxedItems =
+    [
+        new(Amount.Create(20m).Success.Get(), CauseOfExemption.OtherGrounds)
+    ];
+
+    private static readonly TaxRateSummary[] TaxedItems =
+    [
+        GetTaxRateSummary(21m, 42.07M)
```

Comment:
nitpick: naming the params should make it more clear what these numbers mean without diving into the code :)